digits/templates/models/images/classification/new.html

```diff
@@ -647,12 +659,24 @@ <h4 style="display:inline-block;">Python Layers</h4>
                                 <div class="form-group{{mark_errors([form.custom_network])}}">
                                     {{form.custom_network.label}}
                                     {{form.custom_network.explanation(file='/models/images/classification/custom_network_explanation.html')}}
-                                    <div class="pull-right">
-                                        <a href=# class="btn btn-info" onClick="return visualizeNetwork();">Visualize</a>
-                                    </div>
-                                    {{form.custom_network(class='form-control', rows=10)}}
+                                    <div id="divCheckbox" style="display: none;">{{form.custom_network(class='form-control', rows=10)}}</div>
```

Comment:
I was going to suggest that `divCheckbox` be `div_checkbox` or `div-checkbox` as long as it was new, but then saw that it wasn't being used. Is there a reason to id that div?